default/AI/AIDependencies.py

```diff
@@ -88,6 +95,71 @@
 
 PROT_FOCUS_MULTIPLIER = 2.0
 
+# TODO obtain this information from techs.txt
+UNRESEARCHABLE_TECHS = ["SHP_KRILL_SPAWN", "DEF_PLANET_CLOAK"]
+
+UNUSED_TECHS = ["LRN_SPATIAL_DISTORT_GEN", "LRN_GATEWAY_VOID", "LRN_PSY_DOM",
+                "GRO_TERRAFORM", "GRO_BIOTERROR", "GRO_GAIA_TRANS",
+                "PRO_NDIM_ASSMB",
+                "CON_ORGANIC_STRC", "CON_PLANET_DRIVE", "CON_STARGATE",
+                "CON_ART_HEAVENLY", "CON_ART_PLANET",
+                "SHP_NOVA_BOMB", "SHP_DEATH_SPORE", "SHP_BIOTERM"]
+
+THEORY_TECHS = ["LRN_PHYS_BRAIN", "LRN_TRANSLING_THT", "LRN_PSIONICS", "LRN_GRAVITONICS", "LRN_EVERYTHING", "LRN_MIND_VOID", "LRN_NDIM_SUBSPACE", "LRN_TIME_MECH",
+                "GRO_PLANET_ECOL", "GRO_GENETIC_ENG", "GRO_ADV_ECOMAN", "GRO_NANOTECH_MED", "GRO_TRANSORG_SENT",
+                "PRO_NANOTECH_PROD", "PRO_ZERO_GEN",
+                "CON_ASYMP_MATS", "CON_ARCH_PSYCH",
+                "SHP_GAL_EXPLO"]
```

Comment:
Sure, but the problem is there is literally no documentation on how to use that :confused: